datastore/data-loader/client_knmi_station.py

```diff
@@ -19,6 +19,7 @@
 from google.protobuf.timestamp_pb2 import Timestamp
 from parameters import knmi_parameter_names
 
+regex_level = re.compile(r"first|second|third|grass|[0-9]+(\.[0-9]+)?m|(Level )[0-9]+|(?<=\()[ A-z]+(?=\))", re.IGNORECASE)
```

Comment:
Not all of these levels are valid. But they make the parameters uniquely identifiable. So that parameter definitions have a distinguishable ID. But this still results in issues for e.g. /locations when the same parameter has a different description. Although this is less likely to occur because this only happens when a parameter for a different station has a different description as it only takes the latest obs metadata description.